src/Models/QueueMonitor.php

```diff
@@ -94,4 +95,14 @@ public function hasSucceeded(): bool
 
         return ! $this->hasFailed();
     }
+
+    /**
+     * Get the prunable model query.
+     *
+     * @return \Illuminate\Database\Eloquent\Builder
+     */
+    public function prunable()
+    {
+        return static::where('created_at', '<=', now()->subDays(config('filament-jobs-monitor.navigation.retention_days')));
```

Comment:
According to previous suggestion : 
```suggestion
        return static::where('created_at', '<=', now()->subDays(config('filament-jobs-monitor.pruning.retention_days')));
```

Reply:
code edited according to suggection